contracts/MechMarketplace.sol

```diff
@@ -131,8 +132,8 @@ contract MechMarketplace is IErrorsMarketplace {
     mapping(bytes32 => address) public mapPaymentTypeBalanceTrackers;
     // Mapping of account nonces
     mapping(address => uint256) public mapNonces;
-    // Mapping of service ids to mechs
-    mapping(uint256 => address) public mapServiceIdMech;
+    // Mapping of mech to service id
+    mapping(address => uint256) public mapMechServiceIds;
```

Comment:
@DavidMinarsch @mariapiamo this mapping does not make sense anymore IMO, and so is the `remove()` function below.

Each mech service Id is accessible directly via `mech->tokenId()`, and the existence of a mech is validated by the whitelisted factory that created it.

Reply:
OK re removing the mapping.

However, we should still require service id being passed for "create". Otherwise we decouple marketplace from registry, which is not something we wanted to do at this stage. We want to ensure each mech is backed by a service registered on our registry.